hazelcast/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -445,7 +445,7 @@ public void reset() {
                             if (t instanceof CancellationException || t instanceof JetException) {
                                 throw sneakyThrow(t);
                             }
-                            throw new JetException(t.toString(), t);
+                            throw new JetException(t.toString());
```

Comment:
Shouldn't `com.hazelcast.jet.impl.util.ExceptionUtil#stackTraceToString` be used instead of simple `t.toString()` to keep the stacktrace in text form. I think this is what the other case does.

We just need to make sure we don't repeat again that on the new JetException somewhere else.

Reply:
Well, I thought we should fix this place specifically. In general, we have 45 other places where we create `JetException` and pass the cause exception. Should we fix all of them?